client/src/components/utils/Button/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './style.css';
+
+export default function Button({ className, children }) {
+  return (
+    <button type="button" className={`button ${className}`}>
```

Comment:
Make the `className` more specific for the component. I don't recommend using such general name such as `button`
```suggestion
    <button type="button" className={`common-button ${className}`}>
```